src/coreclr/debug/createdump/createdumpmain.cpp

```diff
@@ -199,9 +204,8 @@ int createdump_main(const int argc, const char* argv[])
     {
         if (::GetTempPathA(MAX_LONGPATH, tmpPath) == 0)
         {
-            //printf_error("GetTempPath failed %s", GetLastErrorString().c_str());
             printf_error("GetTempPath failed\n");
-            return ::GetLastError();
+            return -1;
```

Comment:
Losing this information isn't great. Is this bringing complex logic from the pal?

Reply:
Yes, GetLastError() brings complexity from the PAL that I don't want.